### ISSUE TITLE
[MU4] Crescendo shortcut not working on Italian keyboard layout

### ISSUE THREAD

**Issue by sberla365** [musescore/MuseScore#13586]
I am on Linux with Italian keyboard and I cannot trigger crescendo with the default shortcut. The shortcut is defined in Preferences as Shift+, but that combination does not work. When replacing the shortcut with the very same key combination, the program recognises it as the resulting character (which is ; on an Italian keyboard) and everything works well. Decrescendo shortcut is perfectly recognised.
I see that on an American keyboard the default key combinations produce the characters < and > which make more sense and are way more intuitive for crescendo and decrescendo. Being (I believe) < and > universal across languages and keyboard layouts, I think these shortcuts should be defined by the character rather than the key combination, so that it can be more universal. Just a thought since I don't really know the technicality of programming and shortcuts.

I had similar issues with other shortcuts (I cannot remember exactly which one now) which were also defined by the key combination and did not work but they worked when substituted with the very same key combination (also in those cases recognised as the resulting character and not by the combination).

What seems odd to me in all this is that some default shortcuts are defined by key combinations but when adding a custom one it is recognised by the character even when using the very same key combination of the default one.

**Comment by cbjeukendrup** [comment]
> What seems odd to me in all this is that some default shortcuts are defined by key combinations but when adding a custom one it is recognised by the character even when using the very same key combination of the default one.

It seems to me that this is an important problem, and I think it is what I encountered here: https://github.com/musescore/MuseScore/issues/11956#issuecomment-1258304378

**Comment by FrancRos31** [comment]
> The shortcut is defined in Preferences as Shift+, but that combination does not work.
Same issue on Windows (OS: Windows 10 Version 2009 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.2-242570931, revision: github-musescore-musescore-3130f97).

**Comment by krasko78** [comment]
Ciao, ragazzi! I think I have figured it out. Can you check something for me: if you still have `Shift+`, defined for decrescendo, can you clear or assign to something else the following two shortcuts in Preferences -> Shortcuts:

1. Inserisci pausa (TAB)
2. Avanza cursore: tempo seguente (sigla accordale)

and see if Shift+, starts to work?

**Comment by FrancRos31** [comment]
Ciao di nuovo, @krasko78! I've followed your instructions and now the Shift+, shortcut for crescendo works as expected! Seems like there's a shortcuts conflict, both ; and Shift+, share the same key combination (I've already seen your comment about the measure stretch shortcut issue in the German locale). Ottimo lavoro come al solito, grazie!

**Comment by krasko78** [comment]
@FrancRos31 Yeah, definitely the same problem across italian, hungarian, german, etc. keyboards. I'll be working on a permanent fix. Grazie mille!
